src/libraries/Common/tests/Scripts/Tools/ParallelTestExecution.ps1

```diff
@@ -13,7 +13,7 @@
 
 function BuildAndTestBinary
```

Comment:
Who uses this script?

Reply:
no idea. I don't think anyone. Maybe we should delete it...

Reply:
Seems it was used to help stablize networking tests: https://github.com/dotnet/corefx/pull/7937
@dotnet/ncl does your team use this script?

Reply:
ping @dotnet/ncl

Reply:
@ManickaP @scalablecory is the above handle not maintained by your team? Can you answer above's question about who uses this script?

Reply:
Personally, I don't think so, it seems to have been used manually and maybe by some old pipeline.
@wfurt have you ever used this?